### PR TITLE
add SSP2-EU21-EU-Ger-NZ scenario to AMT to regularly test EU/Germany-specific net-zero scenarios

### PR DESCRIPTION
**Pull request by fschreyer**
## Purpose of this PR

This implements https://github.com/remindmodel/development_issues/issues/759. 

Moreover, as the EU21 NPI2025 run got an execution error in the industry post-processing. This has been made more robust. 

Tagging @robertsalzwedel, too. 

## Type of change

*Indicate the items relevant for your PR by replacing* :white_medium_square: *with* :ballot_box_with_check:.\
*Do not delete any lines. This makes it easier to understand which areas are affected by your changes and which are not.*

### Parts concerned
- :ballot_box_with_check: GAMS Code
- :white_medium_square: R-scripts
- :white_medium_square: Documentation (GAMS incode documentation, comments, tutorials)
- :white_medium_square: Input data / CES parameters
- :white_medium_square: Tests, CI/CD (continuous integration/deployment)
- :ballot_box_with_check: Configuration (switches in main.gms, default.cfg, and scenario_config*.csv files)
- :white_medium_square: Other (please give a description)

### Impact
- :white_medium_square: Bug fix
- :white_medium_square: Refactoring
- :ballot_box_with_check: New feature
- :white_medium_square: Change of parameter values or input data (including CES parameters)
- :white_medium_square: Minor change (default scenarios show only small differences)
- :white_medium_square: Fundamental change of results of default scenarios

## Checklist

*Do not delete any line. Leave **unfinished** elements unchecked so others know how far along you are.\
In the end all checkboxes must be ticked before you can merge*.

- [x] **I executed the automated model tests (`make test`) after my final commit and all tests pass (`FAIL 0`)**
- [x] **I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) if and where it was needed**
- [x] **I adjusted the madrat packages (mrremind and other packages involved) for input data generation if and where it was needed**
- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [x] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [x] I updated the `CHANGELOG.md` [correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog) (added, changed, fixed, removed, input data/calibration)

## Further information (optional)

* Runs with these changes are here:
``/p/tmp/schreyer/Modeling/remind/Dev2/output/SSP2-EU21-EU-Ger-NZ_2026-06-16_11.01.16``
* Comparison of results (what changes by this PR?):

### CODE REVIEW

**Comment by robertpietzcker** [comment]
Thanks, Felix!

could you please add to the PR description above
a) a list of specific switch settings that are active in the DEU scenario (compared to a normal Pk750)? I guess you can copy most things over from the issue, but it would be good to have the final list put together here in the PR as well.  

b) move the runs to /p/projects/remind/savePR and make a compScen comparing to a normal EU21 run to show the differences

**Comment by robertsalzwedel** [comment]
Looks good from my side. 

Just some comments: 

I would change the following: 
- `cm_ccsinjeCost <- high` 

Also @VerenaHof and me observed some changes compared to the ARIADNE config, which we wanted to point out. However, I would assume that they are minor points which can also be fixed later: 
- `cm_frac_NetNegEmi` should be set to 0 or default (0.5)? 
- `cm_CESMkup_build` - is that okay to be changed compared to ARIADNE? 
- `cm_exogDem_scen ` - I know we dont want to keep this but how severe do you think are the changes? 
- `cm_ind_energy_limit_manual` misses German specific targets compared to ARIADNE.

File: config/scenario_config.csv [hunk 1]
```diff
@@ -1,53 +1,54 @@
...
...0;;functionalForm;;;100;2055;6;;EUR_regi 0.08, USA_regi 0.07, CHA_regi 0.07, CAZ_regi 0.07,  JPN_regi 0.07, GLO 0.06;DEU.(biodiesel 0.15, bioeths 0.15);transformative;;DEU -0.45;;;;;0;;DEU 0.0025, EU27_regi 0.0025, GLO 0.005;;fehob 1.75, enhb 0.5;manual;2050 . GLO . (ue_cement, ue_steel_primary, ue_steel_secondary)   0.75, 2100 . GLO . (ue_chemicals, ue_otherInd)   0.90, 2100 . DEU . (ue_otherInd, ue_cement, ue_chemicals)   0.01;2050.GLO 0.9;;;2030.2050.EUR_regi.all.year.netGHG_LULUCFGrassi 0.001, 2030.2045.DEU.all.year.netGHG_LULUCFGrassi 0.001;GLO 0.003;;;;;;;;;2;;;se_trade;2035.2045.FRA.DEU.seel 0.183, 2035.2045.ESW.DEU.seh2 0.366, 2035.2045.MEA.DEU.seliqsyn 0.366;;off;off;;DEU 1.3, EU27_regi 7.5;EUR_regi.pebiolc.worldPricemarkup 100;;;Elec_Push;Mix4ICEban;0.1;2030;;SSP2-EU21-NPi2025;;SSP2-EU21-NPi2025;SSP2-EU21-PkBudg750: This climate policy scenario follows the SSP2. The stylized climate policy scenario assumes a peak budget of 750 Gt CO2 on total CO2 emissions from 2015 to 2100. This is a 1.5C scenario, peak warming is allowed to be at or slightly above 1.5C at median climate sensitivity but returns to values below 1.5C in at least 67 % of scenarios by the end of the century.
```
Comment (robertsalzwedel) [inline config/scenario_config.csv:20]: this is missing the increased CCS cost (cm_ccsinjecost = high). 
Otherwise the scenario looks great.

**Review by robertsalzwedel — DISMISSED** [review]
One addition needed then I would approve.

**Comment by fschreyer** [comment]
Thanks for the inputs. 

Here is the list of changes of ``SSP2-EU21-EU-Ger-NZ`` relative ``SSP2-EU21-PkBudg750``:

* ``cm_emiMktTarget = "2030.2050.EUR_regi.all.year.netGHG_LULUCFGrassi 0.001, 2030.2045.DEU.all.year.netGHG_LULUCFGrassi 0.001"``
* ``cm_emiMktTarget_tolerance  GLO 0.003``
* ``cm_bioprod_regi_lim = DEU 1.3, EU27_regi 7.5 ``
* ``cm_maxProdBiolc = off``
* ``cm_import_tax = EUR_regi.pebiolc.worldPricemarkup 100``
* ``c_tech_earlyreti_rate = DEU.(biodiesel 0.15, bioeths 0.15)``
* ``c_ccsinjecrateRegi = DEU 0.0025, EU27_regi 0.0025, GLO 0.005``
* ``c_agricult_base_shift = DEU -0.45``
* ``trade = se_trade`` and ``cm_trade_SE_exog = 2035.2045.FRA.DEU.seel 0.183, 2030.2045.ESW.DEU.seh2 0.366, 2035.2045.MEA.DEU.seliqsyn 0.366``
* ``cm_SEtaxRampUpParam = off``
* ``c_changeProdCost = 0.1`` (reduce cost of deviating from Npi in 2030)
* ``cm_frac_NetNegEmi = 0``
* ``cm_CESMkup_build =fehob 1.75, enhb 0.5`` (prevent the model from continuing oil boilers and adjusting energy efficiency improvements; not sure that is still needed)
* ``cm_ind_energy_limit = manual`` and ``cm_ind_energy_limit_manual = ``2050 . GLO . (ue_cement, ue_steel_primary, ue_steel_secondary)   0.75, 2100 . GLO . (ue_chemicals, ue_otherInd)   0.90, 2100 . DEU . (ue_otherInd, ue_cement, ue_chemicals)   0.01`` (energy efficiency of industry production in Germany same as in NPi; to avoid too fast decline)


I didn't put:

* cm_ccsinjeCost <- high (should have been made default in #2366, but I didn't find the change in the code)
* cm_import_EU = nzero (leads to infes in SSA in 2030 - which must export with this switch - caused by synfuel production capacity bounds in that year)
* ``cm_exogDem_scen``, I think we can live with the default production in the general scenario

Anything I missed? If not, please approve.

**Comment by robertpietzcker** [comment]
> cm_import_EU = nzero (leads to infes in SSA in 2030 - which must export with this switch - caused by synfuel production capacity bounds in that year)

ah, then that is probably still parameterized for the "cm_startyear 2025" version. 
could you update that at some point in time, @Renato-Rodrigues , and then make a new PR to adjust this new DEU-scenario?

**Review by robertpietzcker — APPROVED** [review]
Thanks for adding the summary of all changed switches!

**Comment by Renato-Rodrigues** [comment]
> > cm_import_EU = nzero (leads to infes in SSA in 2030 - which must export with this switch - caused by synfuel production capacity bounds in that year)
> 
> ah, then that is probably still parameterized for the "cm_startyear 2025" version. could you update that at some point in time, @Renato-Rodrigues , and then make a new PR to adjust this new DEU-scenario?

I added to my todo. I will do it in another PR and change the scenario config once I have the time to tackle that.
